consensus-types/types/body.go

```diff
@@ -32,15 +32,14 @@ import (
 )
 
 const (
-	// BodyLengthDeneb is the number of fields in the BeaconBlockBodyDeneb
-	// struct.
+	// BodyLengthDeneb is the number of fields in the BeaconBlockBody struct for Deneb.
 	BodyLengthDeneb uint64 = 12
 
-	// BodyLength is the number of fields in the BeaconBlockBodyElectra struct.
-	BodyLength uint64 = 13
+	// BodyLengthElectra is the number of fields in the BeaconBlockBody struct for Electra.
+	BodyLengthElectra uint64 = 13
 
 	// KZGPosition is the position of BlobKzgCommitments in the block body.
-	KZGPosition = BodyLength - 2
+	KZGPosition uint64 = 11
```

Comment:
unrelated to fork version, this value is always fixed to be 11, so would rather define it absolutely rather than relatively to BodyLength, which is fork version dependent